Feat/update copy (#79)
vacp2p/vac.dev: research-areas.md

---
title: Research areas

> ---

Reviewer: I'd use sembr.

# Overview

Vac has several research areas it is interested in and has or is developing expertise in. Here’s an overview of some of these areas along with motivation and what type of topics are included. Note that a lot of areas are overlapping in terms of specific projects.

## Incentivization

> In a open p2p network, nodes have to be incentivized for services provided to the network. Additionally, these should be privacy-preserving and as trust-minimized as possible. This research areas includes things such as: RLN, service credentials, peer reputation, and protocol revenue.

Reviewer: ```suggestion
In an open p2p network, nodes have to be incentivized for services provided to the network. Additionally, these should be privacy-preserving and as trust-minimized as possible. This research area includes (but is not limited to): RLN, service credentials, peer reputation, and protocol revenue.
```

## Applied ZK

> Applied ZK opens up the door to many new constructs, especially related to privacy. This research area includes things like RLN, Unirep and similar novel constructs, but also things like R&D related to ZKs in general such as Circom, proves on mobile, different proof systems such as Halo 2, etc. Done in close collaboration with other teams in the space.

Reviewer: ```suggestion
Applied ZK opens up the door to many new constructs, especially related to privacy. This research area includes RLN, Unirep and similar novel constructs, but also R&D related to ZKs in general such as Circom, proves on mobile, different proof systems such as Halo 2, etc. This research is conducted in close collaboration with other teams in the space.
```

## Privacy/Anonymity

Specifically metadata privacy. This includes anonymity studies and providing a modular suite of protocols with clear trade-offs and threat models.

## Scalability

> Making the network scalable in terms of number of nodes, messages supported, discovering of nodes, caching of historical messages etc. Largely focused on Waku right now, but things like scalable group chats also relevant here.

Reviewer: ```suggestion
Making the network scalable in terms of number of nodes, messages supported, discovering of nodes, caching of historical messages etc. Largely focused on Waku right now, but things like scalable group chats are also relevant here.
```

## Data synchronization

End to end data application synchronization protocols, FT Store, interfacing with long term storage, message bus guarantees, possibly CRDTs etc. Guiding principle is to provide additional guarantees for users of Waku (etc) in terms of user experience.

## P2P/Networking

> Things related to P2P protocols and networking, including usage of WebRTC, Discovery for resource restricted devices, new p2p protocols with different trade-offs etc.

Reviewer: ```suggestion
R&D related to P2P protocols and networking, including usage of WebRTC,
Discovery for resource restricted devices,
and new p2p protocols with different trade-offs.
```

Reviewer: Imo, the "etc" is implicit in "including"

## Censorship resistance

> Censorship resistance at all layers of the stack, including obfuscated transports, running in hostile environments, etc.

Reviewer: ```suggestion
Censorship resistance at all layers of the stack, including obfuscated transports and running in hostile environments.
```

## Identity/Trust

> Identity and key management required for non-trivial secure messaging. Multi device management, group management, trust, reputation etc. Especially in a private and secure fashion.

Reviewer: ```suggestion
Identity and key management required for non-trivial secure messaging.
This includes multi device management, group management, trust, and reputation.
Especially in a private and secure fashion.
```
